static/app/components/autoComplete.tsx

```diff
@@ -354,7 +344,7 @@ class AutoComplete<T extends Item> extends React.Component<Props<T>, State<T>> {
    *
    * This is exposed to render function
    */
-  closeMenu(...args: Array<any>) {
+  closeMenu = (...args: Array<any>) => {
```

Comment:
ooof, was this broken before?

Reply:
No, but because of how we pass fn down we dont have access to this, so I need to bind it :/